example/javascriptlib/module/5-resources/jest.config.ts

```diff
@@ -1,31 +0,0 @@
-import {pathsToModuleNameMapper} from 'ts-jest';
```

Comment:
Why was this deleted?

Reply:
https://github.com/com-lihaoyi/mill/pull/4398#issuecomment-2614864727